README.md

```diff
@@ -34,34 +34,7 @@ $ java -jar /usr/share/selenium-server/selenium-server-standalone.jar
 $ stack test
 ```
 
-## Overview of the code
-
-### Folder structure
-
-  * `lib` – actual code (as a library)
-  * `src` – just a dumb `Main.hs` file to compile an executable
-  * `tests` – tests
-  * `static` – icons, CSS, Markdown pages, HTML pages, and some JS libraries
-  * `templates` – HTML templates for pages and elements of pages
-  * `scripts` – some scripts used by automatic testing
-  * `favicon` – code used to generate a favicon
-  * `guidejs` – client side JavaScript
-  * `front` – the new frontend, written in Vue.js
-
```

Comment:
Is anywhere frontend structure else?

Reply:
We don't currently have a description of the frontend structure, I will ask the frontend team to have something written in Notion once things slow down (and then we can link the Notion page in the repo).